.eslintrc

```diff
@@ -33,12 +37,45 @@
       "rules": {
         "import/prefer-default-export": 0
       }
+    },
+    {
+      "files": [
+        "tests/**/*.js",
+        "serverless/**/*",
+        "static/src/js/components/**/[^_]*/*.jsx",
+        "static/src/js/**/*.js"
+      ],
+      "rules": {
+        "testing-library/await-async-events": "off",
```

Comment:
Should these be done in the linter package? otherwise EDSC might have a different setup than say MMT. Some of these feel like they aren't bad to ignore though I understand that it would make this PR even larger some comments on why we're doing these would be good to have though

Reply:
I think that is the point of the eslintrc, not all repos will want this, we need to exclude tests that haven't been converted from enzyme etc.